skyportal/utils/offset_util.py

```diff
@@ -0,0 +1,170 @@
+import datetime
+
+import numpy as np
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+from astroquery.gaia import Gaia
+from astropy.time import Time
+
+
+facility_parameters = {'Keck': {"radius_degrees": 2.0/60,
+                                "mag_limit": 18.5,
+                                "mag_min": 11.0,
+                                "min_sep_arcsec": 4.0
+                                },
+                       'P200': {"radius_degrees": 2.0/60,
+                                "mag_limit": 18.0,
+                                "mag_min": 10.0,
+                                "min_sep_arcsec": 5.0
+                                },
+                       'Shane': {"radius_degrees": 2.5/60,
+                                 "mag_limit": 17.0,
+                                 "mag_min": 10.0,
+                                 "min_sep_arcsec": 5.0
+                                 }
+                       }
+
+def get_nearby_offset_stars(source_ra, source_dec, source_name,
```

Comment:
Perhaps just `nearby_offset_stars`?

Reply:
This function is complicated enough that it deserves a test suite.  If you write those up (standard pytest suite), I'd be happy to hook it up to the testing infrastructure so it gets run automatically.

Reply:
It's being tested indirectly via the API call tests.